lib/Conversion/TritonGPUToLLVM/ConvertLayoutOpToLLVM.cpp

```diff
@@ -670,6 +724,13 @@ struct ConvertLayoutOpConversion
       res = SharedToDotOperandMMAv1::convertLayout(
           dotOperandLayout.getOpIdx(), src, smemObj, getThreadId(rewriter, loc),
           loc, getTypeConverter(), rewriter, dst.getType());
+#ifdef USE_ROCM
+      // AMD MI200 matrix cores
+    } else if (!isOuter && mmaLayout.isMI200() && isHMMA) {
+      res = SharedToDotOperandMMAv3::convertLayout(
```

Comment:
I'm sorry to suggest this, but can we refactor `MMAv3` in this PR to MFMA?
Two reasons:
 - MMAv1 and MMAv2 correspond to the 2 versions of Nvidia's mma instructions (Volta, Turing/Ampere) and MFMA is not strictly a v3 of that
 - Presumably at some point in the future there *will* be a v3 of NV's MMA and we'll be in a tough spot when we go to merge.